lambdatimes/src/components/Content/Card.js

```diff
@@ -1,19 +1,31 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
 const Card = props => {
-  return (
-    <div className="card">
-      <div className="headline">{/* headline goes here */}</div>
-      <div className="author">
-        <div className="img-container">
-          <img src={'' /* image source goes here */} />
+    const {card} = props;
+
+    return (
+        <div className="card">
+            <div className="headline">{card.headline}</div>
+            <div className="author">
+                <div className="img-container">
+                    <img src={'' || card.img}/>
```

Comment:
you don't have to do this because `''` (empty string) is always falsy so it will never be that. If anything you can reverse the order and have it as a fallback for `card.img` but for this `<img src={card.img}/>` will be good 👍